app/assets/javascripts/mortgage_calculator/angular/services/stamp_duty.js

```diff
@@ -38,12 +38,20 @@ App.factory('StampDuty', function() {
       cost: function() {
         var totalTax = 0,
             remaining = this.propertyPrice,
-            rates;
+            rates,
+            $conditionalMessage = $('.stamp-duty__FTB_conditional');
 
-        if (this.buyerType === 'isFTB' && this.propertyPrice <= 500000) {
-          rates = this.rates_FTB;
+        if (this.buyerType === 'isFTB') {
```

Comment:
I would move to a function (e.g is_first_time_buyer) that returns true or false. Then the code will be easier to read. Wdyt?

Reply:
Hey @tomas-stefano , I think this is ok because we have multiple conditions which are only used once (isFTB and isSecondHome).  I would recommend renaming the value to 'isFirstTimeBuyer' instead, something for a later date perhaps

Reply:
Ok